docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2090,12 +2090,12 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  If Tomcat Manager is unable to start Tomcat, then it exists the Tomcat.
+  If the Tomcat manager is unable to start itself, then it exits with _code_.
 
   **Action:**
 
-  Review the warning message and see if you are able to check why the tomcat unable to start
-  and exited from tomcat process.
+  Review the warning message and see if you are able to check why the Tomcat unable to start
```

Comment:
Review the message, and if server service is interrupted, send it to support for troubleshooting.